src/cryptsetup/cryptsetup-token-systemd-tpm2/cryptsetup-token.sym

```diff
@@ -0,0 +1,8 @@
+CRYPTSETUP_TOKEN_1.0 {
+    global: cryptsetup_token_open;
+	    cryptsetup_token_buffer_free;
+	    cryptsetup_token_validate;
+	    cryptsetup_token_dump;
+	    cryptsetup_token_version;
+    local: *;
+};
```

Comment:
hmm, it would be best to use the same kind of indentation as src/nss-systemd/nss-systemd.sym for example.